test/evergreen.yml

```diff
@@ -2721,6 +2735,7 @@ buildvariants:
     - name: long-test
     - name: static-wt-build-test
     - name: format-failure-configs-test
+    - name: data-validation-test-checkpoint
```

Comment:
I'm not sure if it's worth being added to the PR testing. How much time does this task run?

Reply:
My suggestion would be to leave it out of PR testing for now. We can always include it later if needed.